Improved resource error handling and translations, improved side button layout, fixed rescan duplicates, and more

click4dylan opened the pull request:
-Improved handling of missing resources so the user knows what file is missing.
-Improved some misspelled english words and changed some text to be more clear
-Added file path to error message when vital resources are missing
-Moved side buttons to the bottom for much better visual clarity

click4dylan committed:
-Improved some misspelled english words and changed some text to be more clear
-Added file path to error message when vital resources are missing
-Moved side buttons to the bottom for much better visual clarity

click4dylan committed:
-Fixed duplicate rom entries being created when rescanning folders
-More english spelling corrections

Speedvicio commented:
I want to congratulate you for having the courage to understand such a messed up source code. 
I still have to test the changes you've made and figure out whether to merge your code or create a dedicated branch on github. thank you for your interest.

Speedvicio committed:
Update MgrSetting.vb

You need to set it to false because whit your changed Resource code MgrSetting will not load the icons
